### PR TITLE
Rectified some Synatx Errors in various py files

### LINKED ISSUE #30: At least three files have Python syntax errors
$ `flake8 . --count --select=E9,F63,F7 --show-source --statistics`
```
./molequeue/python/test/clienttest.py:24:11: E999 SyntaxError: invalid syntax
    print "MoleQueue ID: ", molequeue_id
          ^
./molequeue/molequeue/app/testing/scripts/submitJob.py:19:11: E999 SyntaxError: invalid syntax
    print "Client %s connecting to socket: %s"%(clientId, socketname)
          ^
./molequeue/molequeue/app/testing/scripts/sendRpcKill.py:13:11: E999 SyntaxError: invalid syntax
    print "Connecting to socket: %s"%socketname
          ^
3     E999 SyntaxError: invalid syntax
3
```

### PR DESCRIPTION
**Pull request by adityaomar3**
There were few synatx errors in py file mostly due to change in version of python. Rectified in this commit

### CODE REVIEW

File: molequeue/app/testing/scripts/sendRpcKill.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 def run_test():
   global socketname
   if debug:
-    print "Connecting to socket: %s"%socketname
+    print ("Connecting to socket: %s"%socketname)
```
Comment (cclauss) [inline molequeue/app/testing/scripts/sendRpcKill.py]: Correct.  `print()` is a function in Python 3 but write it like other functions as `print()`, not `print ()`...
```suggestion
    print("Connecting to socket: %s"%socketname)
```

File: molequeue/app/testing/scripts/sendRpcKill.py [hunk 2]
```diff
@@ -32,10 +32,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print ex
+    print (ex)
```
Comment (cclauss) [inline molequeue/app/testing/scripts/sendRpcKill.py]:
```suggestion
    print(ex)
```

File: molequeue/app/testing/scripts/sendRpcKill.py [hunk 3]
```diff
@@ -32,10 +32,10 @@ def main(argv):
...
     test_result = 1
   if debug:
-    print "Exiting with code: %d"%test_result
+    print ("Exiting with code: %d"%test_result)
```
Comment (cclauss) [inline molequeue/app/testing/scripts/sendRpcKill.py]:
```suggestion
    print("Exiting with code: %d" % test_result)
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 4]
```diff
@@ -16,7 +16,7 @@
 def run_test():
   global socketname, molequeue_ids_done, clientId, mq_id_lock, num_jobs
   if debug:
-    print "Client %s connecting to socket: %s"%(clientId, socketname)
+    print ("Client %s connecting to socket: %s"%(clientId, socketname))
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
    print("Client %s connecting to socket: %s" % (clientId, socketname))
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 5]
```diff
@@ -29,9 +29,9 @@ def notification_callback(msg):
           with mq_id_lock:
             molequeue_ids_done.append(moleQueueId)
           if debug:
-            print "Job %d finished! (Client %s)"%(moleQueueId, clientId)
+            print ("Job %d finished! (Client %s)"%(moleQueueId, clientId))
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
            print("Job %d finished! (Client %s)" % (moleQueueId, clientId))
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 6]
```diff
@@ -29,9 +29,9 @@ def notification_callback(msg):
...
     except Exception as ex:
-      print "Unexpected notification:", msg, ex
+      print ("Unexpected notification:", msg, ex)
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
      print("Unexpected notification:", msg, ex)
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 7]
```diff
@@ -50,15 +50,15 @@ def notification_callback(msg):
       client.disconnect()
       raise Exception("Connection timed out!")
     if debug:
-      print "Submitted job %d (Client %s)"%(molequeue_id, clientId)
+      print ("Submitted job %d (Client %s)"%(molequeue_id, clientId))
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
      print("Submitted job %d (Client %s)" % (molequeue_id, clientId))
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 8]
```diff
@@ -50,15 +50,15 @@ def notification_callback(msg):
...
     sys.stdout.flush()
 
   timeout = 30
   mq_id_lock.acquire()
   while len(molequeue_ids) != len(molequeue_ids_done) and timeout > 0:
     if debug:
-      print "Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
-        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done))
+      print ("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
+        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done)))
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)" %
        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done)))
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 9]
```diff
@@ -90,10 +90,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print ex
+    print (ex)
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
    print(ex)
```

File: molequeue/app/testing/scripts/submitJob.py [hunk 10]
```diff
@@ -90,10 +90,10 @@ def main(argv):
...
     test_result = 1
   if debug:
-    print "Exiting with code: %d (client %s)"%(test_result, clientId)
+    print ("Exiting with code: %d (client %s)"%(test_result, clientId))
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
    print("Exiting with code: %d (client %s)" % (test_result, clientId))
```

File: python/test/clienttest.py [hunk 11]
```diff
@@ -21,7 +21,7 @@ def test_submit_job(self):
 
     molequeue_id = client.submit_job(job)
 
-    print "MoleQueue ID: ", molequeue_id
+    print ("MoleQueue ID: ", molequeue_id)
```
Comment (cclauss) [inline python/test/clienttest.py]:
```suggestion
    print("MoleQueue ID: ", molequeue_id)
```

File: python/test/clienttest.py [hunk 12]
```diff
@@ -105,7 +105,7 @@ def test_request_queue_list_update(self):
     queues = client.request_queue_list_update()
 
     for q in queues:
-      print q.name, ", ", q.programs;
+      print (q.name, ", ", q.programs)
```
Comment (cclauss) [inline python/test/clienttest.py]:
```suggestion
      print(q.name, ", ", q.programs)
```

**Review by cclauss — CHANGES_REQUESTED** [review]

File: molequeue/app/testing/scripts/sendRpcKill.py (hunk 1 above)
Comment (adityaomar3) [reply molequeue/app/testing/scripts/sendRpcKill.py]: > Correct. `print()` is a function in Python 3 but write it like other functions as `print()`, not `print ()`...

Yeah sure, that may have happened due to auto formatting of code

Comment (cclauss) [reply molequeue/app/testing/scripts/sendRpcKill.py]: I see no such automation...  https://github.com/OpenChemistry/molequeue/actions

**Comment by adityaomar3** [comment]
@cclauss successfully removed the additional spaces, please review it again

File: molequeue/app/testing/scripts/sendRpcKill.py (hunk 1 above)
Comment (adityaomar3) [reply molequeue/app/testing/scripts/sendRpcKill.py]: > I see no such automation... https://github.com/OpenChemistry/molequeue/actions

Sorry I meant by VS code in my pc so when I pushed the spaces remained but now I have rectified it.

File: molequeue/app/testing/scripts/submitJob.py [hunk 13]
```diff
@@ -50,15 +50,15 @@ def notification_callback(msg):
...
+      print("Submitted job %d (Client %s)"%(molequeue_id, clientId))
     sys.stdout.flush()
 
   timeout = 30
   mq_id_lock.acquire()
   while len(molequeue_ids) != len(molequeue_ids_done) and timeout > 0:
     if debug:
-      print "Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
-        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done))
+      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
```
Comment (cclauss) [inline molequeue/app/testing/scripts/submitJob.py]:
```suggestion
      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%
```

Comment (adityaomar3) [reply molequeue/app/testing/scripts/submitJob.py]: > 

Done

**Review by cclauss — APPROVED** [review]

**Comment by adityaomar3** [comment]
@cclauss @cryos can we now merge this to master?
is this project dead @cryos

**Comment by ghutchis** [comment]
@adityaomar3 - it's not quite dead, but it's been accumulating rust.

I'll take a look at this in the next day or two.

**Pull request merged**